py/torch_tensorrt/dynamo/_defaults.py

```diff
@@ -11,3 +12,4 @@
 TRUNCATE_LONG_AND_DOUBLE = False
 USE_PYTHON_RUNTIME = False
 USE_FAST_PARTITIONER = True
+DEVICE = Device(gpu_id=torch.cuda.current_device())
```

Comment:
Thinking about this, it might not be best to have the default here since this code will only execute once iirc. Better to just make it the default of the device class